XU-TechnicalPaper.md

```diff
@@ -102,17 +102,60 @@ Payment channel atomic swap technology is fairly new and was just recently succe
+
+Sidenote: Transferring order execution authority to a third party, in particular to a decentralized consensus, was considered and deemed not feasible for the following reasons:
+a) consensus=slow
+b) hard to sell; an exchange won't let anyone else decide on who fills its users orders
+c) hard to control since in our case exchanges could always treat their local order book with priority
+
+3. Reward liquidity providers
+Makers are earning XUC, takers are paying in XUC for filling orders. Makers set a price for each order in a specified fee field. We aim to establish a fee market where makers are competing with a combination of order quantity, price and XUC fee. A taker pays the full XUC fee via a default Raiden Channel to the maker. It's down to exchanges to take over the role of handling XUC payments and earnings for their users or integrate this as a business model for traders. 
+
+4. Punish malicious behavior
+Market makers are required to stake a certain amount of XUC in a smart contract in order to be able to act as a market maker. This will be used to execute punishments.
+
+5. Privacy between two trading parties (not part of PoC)
+As it is the case on regular digital asset exchanges, a maker and a taker should have the option to remain anonymous to avoid biased behavior and preserve privacy. On payment channels this is taken care of by [onion routing](https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md). A tor-hidden service for order book information exchange is currently being explored, but is not a priority for the PoC for now.
+ 
+It is evident, that the approach to enforce direct socket connections between peers to exchange order book information can't scale infinitely. We are targeting to establish a 'super-node' approach in a later phase, where entities running an XU node can choose to forward and receive order book information from this node. To become a supernode for a peer, it has to provide a faster connection to a specific other peer as minimal requirement. This can be achieved, for example through a strategic location on a fibre connection, where peers wouldn't have direct access to and help to further scale and speed-up order book information exchange.
```

Comment:
You can use the term 'relay network' to describe the 'super-node' approach.

Reply:
Like that! Done.